src/librustc_resolve/macros.rs

```diff
@@ -469,4 +480,38 @@ impl<'a> Resolver<'a> {
             self.exported_macros.push(def);
         }
     }
+
+    /// Error if `ext` is a Macros 1.1 procedural macro being imported by `#[macro_use]`
+    fn err_if_macro_use_proc_macro(&mut self, name: Name, use_span: Span, binding: &NameBinding) {
```

Comment:
needs to be `binding: &NameBinding<'a>` to avoid the lifetime error in travis failure.